nuget/helpers/lib/NuGetUpdater/Directory.Common.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <CommonTargetFramework>net8.0</CommonTargetFramework>
```

Comment:
Can we add a note that when updating this property, we should also ensure that we also have updated the [SupportedFrameworks](https://github.com/dependabot/dependabot-core/blob/main/nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/FrameworkChecker/SupportedFrameworks.cs) used be the framework compatibility check? It should at least follow the changes made to [NuGetGallery](https://github.com/NuGet/NuGetGallery/blob/main/src/NuGetGallery.Core/Frameworks/SupportedFrameworks.cs), although ideally be ahead.

Reply:
Good catch, I missed updating the supported frameworks check.  I'll update that and add a comment.